### PR TITLE
:recycle: refactor: remake of info-card

### PR DESCRIPTION
**Pull request by atilasousa**

### CODE REVIEW

File: src/lib/components/info-card/index.tsx [hunk 1]
```diff
@@ -16,28 +16,37 @@ export const InfoCard: FC<PropsWithChildren<InfoCardProps>> = ({
 
   return (
     <Box
-      p="m"
-      width={width ?? '15rem'}
-      height="7.5rem"
+      px="m"
+      py="xl"
+      bg="onPrimary"
+      height="8.25rem"
```
Comment (KipandaJr) [inline src/lib/components/info-card/index.tsx:22]: try remove this `height` and put a `gap="l or m"` (according theme-space to 1rem)

File: src/lib/components/info-card/index.tsx [hunk 2]
```diff
@@ -16,28 +16,37 @@ export const InfoCard: FC<PropsWithChildren<InfoCardProps>> = ({
...
       borderRadius="m"
       overflow="hidden"
-      border="1px dashed"
       display="inline-flex"
       flexDirection="column"
+      width={width ?? '26.75rem'}
```
Comment (KipandaJr) [inline src/lib/components/info-card/index.tsx:27]: instead use `with` try put a `max-width`

File: src/lib/components/info-card/index.tsx [hunk 3]
```diff
@@ -16,28 +16,37 @@ export const InfoCard: FC<PropsWithChildren<InfoCardProps>> = ({
...
       justifyContent="space-between"
-      borderColor="outline.outlineVariant"
       color={dark ? 'white' : 'onSurface'}
```
Comment (KipandaJr) [inline src/lib/components/info-card/index.tsx:29]: remove color `white` and put `lowestContainer` to white color

File: src/lib/components/info-card/index.tsx [hunk 4]
```diff
@@ -16,28 +16,37 @@ export const InfoCard: FC<PropsWithChildren<InfoCardProps>> = ({
...
+          bg="surface"
+          padding="2xs"
+          width="2.5rem"
+          display="flex"
+          height="2.5rem"
+          borderRadius="50%"
+          alignItems="center"
+          justifyContent="center"
+        >
+          <MoneySignSVG maxHeight="1.5rem" maxWidth="1.5rem" width="100%" />
```
Comment (KipandaJr) [inline src/lib/components/info-card/index.tsx:46]: Icon should be a props, `ReactNode` prop

File: src/lib/components/info-card/info-card.types.ts [hunk 5]
```diff
@@ -3,7 +3,6 @@ import { CSSProperties, ReactNode } from 'react';
 type MaybeArray<T> = ReadonlyArray<T> | T;
 
 export interface InfoCardProps {
-  info: ReactNode;
   title: ReactNode;
```
Comment (KipandaJr) [inline src/lib/components/info-card/info-card.types.ts:6]: put one props to icon like `icon: ReactNode`

File: src/lib/components/info-card/index.tsx
Comment (KipandaJr) [inline src/lib/components/info-card/index.tsx:1]: see when we put a custom `width`, i think we need a `min-width` ou put scroll bar when its so short, `Overflow-x` in this case. cuz when we put a so short value the card have a weird behavior

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image

the padding Y and X should be equal according figma

**Review by MarioBatalha — APPROVED** [review]

File: src/lib/components/info-card/info-card.types.ts [hunk 6]
```diff
@@ -3,7 +3,7 @@ import { CSSProperties, ReactNode } from 'react';
 type MaybeArray<T> = ReadonlyArray<T> | T;
 
 export interface InfoCardProps {
-  info: ReactNode;
+  icon?: ReactNode;
```
Comment (KipandaJr) [inline src/lib/components/info-card/info-card.types.ts:6]: React Node Props need start capitalized

**Comment by git-marcopitra** [comment]
image

Take a look at info card footer on Storybook

**Review by git-marcopitra — CHANGES_REQUESTED** [review]
See Antonio's changes

File: pnpm-lock.yaml
Comment (git-marcopitra) [inline pnpm-lock.yaml:1]: remove this file

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Pull request merged**